### DISCUSSION TITLE
Running Dangerous executable as Administrator within Sandboxie - Safe?

### DISCUSSION THREAD

**Discussion by kimpurcell** [sandboxie-plus/Sandboxie#1386] [Q&A Feedback] [answered]
I have a shortcut to launch an executable under a Sandboxie locked down sandbox.

If I launch that shortcut with an Administrator account, can the executable circumvent/escape the sandbox since it was run by an Admin account?

**Comment by DavidXanatos** [comment, marked as answer]
It should not be able, but it will have more opportunity to exploit a potentially present unknown vulnerability.
